packages/react-dom/src/events/DOMModernPluginEventSystem.js

```diff
@@ -896,6 +909,7 @@ export function accumulateEventTargetListeners(
   dispatchQueue: DispatchQueue,
```

Comment:
So this is only used by the new EventHandle code? Can we rename this to `accumulateEventHandleListeners` or something like it? It took me a few reads to notice this wasn't actually used by the regular event system. The "event target" name looked generic to me.

Reply:
Renamed to `accumulateEventHandleTargetListeners`.